securing_apps/topics/oidc/oidc-generic.adoc

```diff
@@ -200,6 +209,16 @@ If user authentication is complete, the application obtains the device code. The
 
 For more details refer to the https://tools.ietf.org/html/rfc8628[OAuth 2.0 Device Authorization Grant specification].
 
+===== Client Initiated Backchannel Authentication Grant
+
+Client Initiated Backchannel Authentication Grant is used by clients who want to initiate the authentication flow by communicating with the OpenID Provider directly without redirect through the user's browser like OAuth 2.0's authorization code grant.
+
+The client requests {project_name} an auth_req_id that identifies the authentication request made by the client. {project_name} creates the auth_req_id.
+
+After receiving this auth_req_id, this client repeatedly needs to poll {project_name} to obtain an Access Token, Refresh Token and ID Token from {project_name} in return for the auth_req_id until the user is authenticated. 
+
+For more details refer to https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html[OpenID Connect Client Initiated Backchannel Authentication Flow specification].
```

Comment:
Again, I suggest to add cross-linking and attach some additional info sentence like this:
`Also please refer to other places of {project_name} documentation like <Link to "Backchannel Authentication Endpoint" of this securing_apps guide> and <Client Initiated Backchannel Authentication Grant of server admin guide> .`

Reply:
Yes, I would like to add such a cross-linking sentence.

Reply:
+1, Thanks!